### PR TITLE
Grammar improvements to the documentation

### PR DESCRIPTION
**Pull request by MinganMuon**
Here's some improvements to the grammar of the documentation.

I was just passing through, looking at repos from GitHub Explore, when I noticed some missing articles and just had to fix them. I'm an editor at heart. :)

I've made some comments in the commit on edits I'm not certain about; please do take a look.

Disclaimer: I've never played FFVIII or worked on/played with this project (not yet anyway), so if one of my changes alters the meaning of the clause under concern in a way that contradicts with the prior meaning, I may not have noticed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,31 +51,31 @@ Requirements: MonoGame + Visual Studio
 
 1. Download and install **Visual Studio 2017** (2015 is not supported) and **NET Framework 4.6**
 
-2. Clone whole repository:
+2. Clone the whole repository:
```
Comment (eve-atum) [inline README.md]: Actually, "Clone the repository:" would be better

File: README.md [hunk 2]
```diff
@@ -51,31 +51,31 @@ Requirements: MonoGame + Visual Studio
...
 
-6. Make sure you add Final Fantasy VIII path to array at `WindowsGameLocationProvider.cs:36`. On Windows the code tries to detect the install path via the registry. If it fails, it'll fall back to the array.
+6. Make sure you add the Final Fantasy VIII path to the array at `WindowsGameLocationProvider.cs:36`. On Windows the code tries to detect the install path via the registry. If it fails, it'll fall back to the array.
 
 7. That's all. You can now compile the executable.
 
 ## Getting started (Linux/Mono) [Tested on Ubuntu]
 
-1. Make sure your Linux is up to date. Due to FFmpeg dependency we require Ubuntu Cosmos
+1. Make sure your Linux is up to date. Due to the FFmpeg dependency we require Ubuntu Cosmos.
```
Comment (eve-atum) [inline README.md]: "Due to ffmpeg dependencies, we require Ubuntu Cosmos."

File: README.md [hunk 3]
```diff
@@ -107,13 +107,13 @@ Requirements: MonoGame + Visual Studio
...
 
-9. Make sure you add Final Fantasy VIII path to array at `LinuxGameLocationProvider.cs:18`
+9. Make sure you add the Final Fantasy VIII path to the array at `LinuxGameLocationProvider.cs:18`
 
 10. That's all. You can now compile the executable.
 
 ## Development guidelines
 
-1. Project is in in-dev prototype, therefore you can make new pull requests directly to main branch. 
+1. Project is an in-dev prototype, therefore you can make new pull requests directly to main branch.
```
Comment (eve-atum) [inline README.md]: 'in' in this case refers to a project status, rather than a product.

File: docs/README.md [hunk 4]
```diff
@@ -2,48 +2,48 @@
...
 
 ## Is Steam version enough to play with OpenVIII? Can I use PC2000 version?
 
 Yes. The Steam release is the best option to be used with OpenVIII. Other than that we also fully support the first PC release.
 
 ## Which game languages are available?
 
-Any that your game works on. OpenVIII is a game engine, we do not have any kind of text in our assets. Everything is read directly from your game catalogue. If you installed French version of the game, then OpenVIII will display the game in French.
+Any that your game works on. OpenVIII is a game engine: we do not have any kind of text in our assets. Everything is read directly from your game catalogue. If you installed French version of the game, then OpenVIII will display the game in French.
```
Comment (eve-atum) [inline docs/README.md]: comma or semi-colon, not full colon.

File: README.md (hunk 1 above)
Comment (MinganMuon) [reply README.md]: True.

File: README.md (hunk 2 above)
Comment (MinganMuon) [reply README.md]: But it's a single dependency on the ffmpeg library, not multiple dependencies.

File: README.md (hunk 3 above)
Comment (MinganMuon) [reply README.md]: True, and I think that "in" would be fine if it was written "Project is in dev", but here "in-dev" is an adjective modifying the noun "prototype", so we need an article.

File: docs/README.md (hunk 4 above)
Comment (MinganMuon) [reply docs/README.md]: Changing to semicolon then.

File: README.md (hunk 3 above)
Comment (MinganMuon) [reply README.md]: Maybe "This project is currently in active development, therefore..." would be better?

**Pull request merged**